tests/uricontent-01/test.yaml

```diff
@@ -0,0 +1,22 @@
+# *** Add configuration here ***
+
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
```

Comment:
We probably don't need the `stats` event but it doesn't hurt without the checks for other params, I think.